setup.py

```diff
@@ -131,6 +145,7 @@ def run(self):
 
 cmdclass['build'] = css_first(build)
 cmdclass['sdist'] = css_first(sdist)
+cmdclass['bdist_egg'] = bdist_egg if 'bdist_egg' in sys.argv else bdist_egg_disabled
```

Comment:
@minrk - just noticed that this and Jupyterlab both have a conditional to allow explicit building of eggs. Is there a reason that's important, or are we just anticipating the possibility that someone may want it?